jcl/src/java.base/share/classes/com/ibm/oti/util/ExternalMessages-MasterIndex.properties

```diff
@@ -638,7 +638,7 @@ K0378=no key for cert chain
 K0379=failed to import key into cl3
 K037a=the data to be signed cannot be an empty string
 K037b=Must be in bootstrap class path to call this method
-K037c=Mutex allready exists
+K037c=Mutex already exists
```

Comment:
This line can just be deleted, it isn't used. If you don't want to do it, I can create another PR.

Reply:
I'd rather not do it in this PR. If you're ok w/ splitting it, that'd be great.

Reply:
https://github.com/eclipse/openj9/pull/5958